website/Dockerfile

```diff
@@ -29,6 +29,8 @@ ENV DEBIAN_FRONTEND=noninteractive \
     LC_CTYPE=C.UTF-8 \
     LC_MESSAGES=C.UTF-8
 
+WORKDIR /opt/
```

Comment:
Why this change?

Reply:
Without moving this earlier, npm fails with an otherwise inscrutable "ideal tree" error. This was the reported solution (setting a workdir, so in this case, it's set earlier).